temp_library/helpers.py

```diff
@@ -3,117 +3,103 @@
 
 
 def openscrn():  # for my personal game-change to your liking

```

Comment:
A helper function can not have a specific value. Pass it through as a parameter. Add the specific code in the example.